Overview.src.html

```diff
@@ -3026,6 +3080,45 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
  to `<code title>GET</code>` and <var>request</var>'s <span title=concept-request-body>body</span>
  to null.
 
+ <li><p>Let <var>useReferredTokenBinding</var> be the result of
+   <span title=concept-header-parse>parsing</span>
+   `<code title>Include-Referred-Token-Binding-ID</code>` in <var>actualResponse</var>'s
+   <span title=concept-response-header-list>header list</span>.
+   The value of this <span title=concept-header>header</span> is specified in
+   <a href="https://tools.ietf.org/id/draft-ietf-tokbind-https-05.xml#rfc.section.3.3">section 3.3</a>
+   of the Token Binding over HTTP spec.
```

Comment:
Is there some kind of latest draft link we could use or will you update Fetch each time a new draft is issued?